tpchgen/src/generators.rs

```diff
@@ -23,16 +23,21 @@ pub struct NationGenerator<'a> {
 
 impl Default for NationGenerator<'_> {
     fn default() -> Self {
-        Self::new()
+        // arguments are ignored
+        Self::new(1.0, 1, 1)
     }
 }
 
 impl<'a> NationGenerator<'a> {
     /// Creates a new NationGenerator with default distributions and text pool
     ///
+    /// Nations does not depend on the scale factor or the part number. The signature of
+    /// this method is provided to be consistent with the other generators, but the
+    /// parameters are ignored.
+    ///
     /// The generator's lifetime is `&'static` because it references global
     /// [`Distribution]`s and thus can be shared safely between threads.
-    pub fn new() -> NationGenerator<'static> {
+    pub fn new(_scale_factor: f64, _part: i32, _part_count: i32) -> NationGenerator<'static> {
```

Comment:
I made these methods take the same arguments as the other generators to simplify the macros (all generators now have the same signature)